unicon_backend/evaluator/tasks/programming/runner.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any
+
+from unicon_backend.evaluator.tasks.base import Task, TaskEvalResult, TaskEvalStatus
+from unicon_backend.evaluator.tasks.programming.base import ProgrammingTask
+from unicon_backend.workers.task_publisher import task_publisher
+
+
+def run_programming_task(task: ProgrammingTask, input: Any) -> TaskEvalResult:
+    # NOTE: It is safe to ignore type checking here because the type of task is determined by the "type" field
+    # As long as the "type" field is set correctly, the type of task will be inferred correctly
+    runner_job = task.create_runner_job(task.validate_user_input(input))  # type: ignore
+    task_publisher.publish_runner_job(runner_job)
+    return TaskEvalResult(task_id=task.id, status=TaskEvalStatus.PENDING_PUSH, result=runner_job.id)
+
+
+def run_as_programming_task(task: Task, input: Any) -> TaskEvalResult:
```

Comment:
is the plan to have `run_as_X_task` moving forward?

Reply:
I’m considering updating the `TaskOrm.to_task` method in the future so that it returns the appropriate task subclass, as part of a broader backend refactoring effort I have in mind. For now, though, my plan is to keep changes minimal and avoid major modifications to the existing codebase.